autovalet_system/launch/state_machine.launch

```diff
@@ -31,7 +31,7 @@
 
         <!-- STATE MACHINE -->
         <!-- state machine node runs lane detection, goal generation, and parking subsystems, as well as overall system logic -->
-        <node name="STATE_MACHINE" pkg="autovalet_system" type="run_sm.sh" args="$(arg simulation)" />
+        <!--node name="STATE_MACHINE" pkg="autovalet_system" type="run_sm.sh" args="$(arg simulation)" /-->
```

Comment:
not sure if we wanna have the state machine launching script commented or not in the final version. this line calls the shell script that does bootup. we manually did this call for FVD

Reply:
I think we should leave it uncommented no that I think about it. 'cause it was intended to be an automatic system with one ENTER hit?

Reply:
@SubramanianKrish  opinion?

Reply:
we should probably comment it out. It's because we never tried the one press to rule them all system on the hw, so if anyone tries to run the code (which I don't expect in a million years to happen), the bash run version shouldn't break other things. Bootup of nodes and packages are different in sim vs hw, so it's better to have it commented for the hw as it hasn't been tested.

Reply:
I see there's a logic branch here. For sim, I suggest uncommenting it. for he comment it out.

Reply:
i like this ^